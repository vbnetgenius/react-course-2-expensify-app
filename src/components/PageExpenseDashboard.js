import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import { connect } from 'react-redux';

const PageExpenseDashboard = () => (
    <div>
        <ExpenseListFilters />
        <ExpenseList />
    </div>
);

export default PageExpenseDashboard;