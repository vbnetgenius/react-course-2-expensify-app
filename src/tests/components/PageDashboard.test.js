import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import PageExpenseDashboard from './../../components/PageExpenseDashboard';

test('should render PageDashboard correctly', () => {
    const wrapper = shallow(<PageExpenseDashboard />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});