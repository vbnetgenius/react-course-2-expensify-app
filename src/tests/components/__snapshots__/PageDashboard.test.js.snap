// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PageDashboard correctly 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
